### PR TITLE
Integrate optional actions w/ library

### PR DESCRIPTION
**Pull request by zerotucks**
NOTE: Breaks create/close position tasks as they fetch the deployed version of the Operation Executor from mainnet

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,7 +65,7 @@
     "ethereum-waffle": "^3.1.1",
     "ethers": "^5.0.19",
     "googleapis": "107",
-    "hardhat": "2.9.5",
+    "hardhat": "2.8.0",
```
Comment (fenris85) [inline package.json:68]: why downgrading hardhat?

File: packages/oasis-actions/src/operations/aave/close.ts [hunk 2]
```diff
@@ -93,9 +100,12 @@ export async function close(
...
       withdrawDAIFromAAVE,
       unwrapEth,
-      returnFunds,
+      returnDebtFunds,
+      returnCollateralFunds,
     ],
   })
 
-  return { calls: [takeAFlashLoan], operationName: 'CUSTOM_OPERATION' }
+  const operationName = OPERATION_NAMES.aave.CLOSE_POSITION
```
Comment (fenris85) [inline packages/oasis-actions/src/operations/aave/close.ts]: Is assigning to a new variable needed here? If so, then you can do `return { operationName, calls: [takeAFlashLoan] }`

File: packages/oasis-actions/src/operations/aave/decreaseMultiple.ts [hunk 3]
```diff
@@ -74,21 +76,25 @@ export async function decreaseMultiple(
...
-      withdrawDAIFromAAVE,
-    ],
+    calls: flashloanCalls,
   })
 
-  return { calls: [takeAFlashLoan], operationName: 'CUSTOM_OPERATION' }
+  const calls = [takeAFlashLoan]
+  const operationName = OPERATION_NAMES.aave.DECREASE_POSITION
+
+  return { calls, operationName }
```
Comment (fenris85) [inline packages/oasis-actions/src/operations/aave/decreaseMultiple.ts]: Here it's better ;),  but I would still combine it to single return

File: packages/oasis-actions/src/operations/aave/increaseMultiple.ts [hunk 4]
```diff
@@ -127,11 +127,10 @@ export async function increaseMultiple(
   })
 
   const calls = [takeAFlashLoan]
-  use.pullDebtTokensInToProxy && calls.unshift(pullDebtTokensToProxy)
-  use.pullCollateralInToProxy && calls.unshift(pullCollateralTokensToProxy)
+  const operationName = OPERATION_NAMES.aave.INCREASE_POSITION
```
Comment (fenris85) [inline packages/oasis-actions/src/operations/aave/increaseMultiple.ts]: same as above

File: package.json (hunk 1 above)
Comment (zerotucks) [reply package.json:68]: tenderly export support. 2.9.5 doesn't work with tenderly. And the only think 2.8 breaks is Maker tests which don't seem relevant right now.

So, perhaps we leave at 2.8.0 so people don't have to keep downgrading during development? And then address Maker tests once they become relevant again?

**Review by jakubswierczek — APPROVED** [review]

**Pull request merged**